bigtable/CMakeLists.txt

```diff
@@ -108,13 +110,15 @@ add_library(bigtable_client
     client/mutations.h
     client/mutations.cc
     client/row.h
+    client/row_range.h
+    client/row_range.cc
     client/rpc_backoff_policy.h
     client/rpc_backoff_policy.cc
     client/rpc_retry_policy.h
     client/rpc_retry_policy.cc
     client/table.h
     client/table.cc
-    client/version.h)
+    client/version.h client/internal/row_key_compare.cc client/internal/row_key_compare.h)
```

Comment:
Why are some files on their own line, and some on a line with many other files?

Reply:
Fixed.